src/www/ui/ui-view-info.php

```diff
@@ -116,13 +116,17 @@ function ShowView($Upload, $Item, $ShowMenu=0)
       } else {
         $bytesH = '(' . $bytesH . ')';
       }
+      $sql_sh = "select * from software_heritage where pfile_fk = $1";
+      $row_sh = $this->dbManager->getSingleRow($sql_sh,array($row['pfile_fk']),
+            __METHOD__."GetSoftwareHeritageRow");
```

Comment:
This code can be removed. use existing function getSoftwareHetiageRecord($pfileId) from src/lib/php/Dao/SoftwareHeritageDao.php to get the required data.

Reply:
Yes I am adding that now.